molgenis/client.py

```diff
@@ -118,7 +124,8 @@ def get(self,
             sort_column: str = None,
             sort_order: str = None,
             raw: bool = False,
-            expand: str = None) -> Union[List[dict], dict]:
+            expand: str = None,
+            uploadable: bool = False) -> Union[List[dict], dict]:
```

Comment:
Add a line to the docstring explaining what it does

Reply:
Have you considered adding an `uploadable` parameter to the `get_by_id()` method?

Reply:
Both done